### PR TITLE
`1deg_jra55do_ryf`: update PE layout

### PR DESCRIPTION
**Pull request by minghangli-uni**
See https://github.com/COSIMA/access-om3/issues/130#issuecomment-2301146030

### CODE REVIEW

File: nuopc.runconfig [hunk 1]
```diff
@@ -40,31 +40,31 @@ PELAYOUT_attributes::
...
+     lnd_ntasks = 1
      lnd_nthreads = 1
      lnd_pestride = 1
      lnd_rootpe = 0
      ninst = 1
-     ocn_ntasks = 48
+     ocn_ntasks = 216
      ocn_nthreads = 1
      ocn_pestride = 1
      ocn_rootpe = 0
```
Comment (dougiesquire) [inline nuopc.runconfig:59]:
```suggestion
     ocn_rootpe = 24
```

**Review by dougiesquire — CHANGES_REQUESTED** [review]
`ocn_rootpe` needs to be modified as per my suggestion below.

Also the `config.yaml` needs to be changed as in [my original comment](https://github.com/COSIMA/access-om3/issues/130#issuecomment-2301146030)

File: nuopc.runseq [hunk 2]
```diff
@@ -1,5 +1,5 @@
 runSeq:: 
-@1800
+@3600
```
Comment (dougiesquire) [inline nuopc.runseq:2]: I don't understand why this change is here after rebasing? Would you like me to rebase this branch on `1deg_jra55do_ryf`?

**Review by dougiesquire — CHANGES_REQUESTED** [review]
Still not quite right. e12a794a2bfda41173fd101acd47d4d52681a403 should not be in this PR since it's already included in `1deg_jra55do_ryf`.

To rebase you need to:
```
git checkout 1deg_jra55do_ryf
git pull
git checkout 1deg_jra55do_ryf.iss130
git rebase 1deg_jra55do_ryf
```

**Comment by minghangli-uni** [comment]
I forgot https://github.com/COSIMA/MOM6-CICE6/commit/e12a794a2bfda41173fd101acd47d4d52681a403 was merged. Now it should be right.

**Review by dougiesquire — APPROVED** [review]
Thanks @minghangli-uni. LGTM

**Comment by minghangli-uni** [comment]
!cherry-pick f3d4e7d into 1deg_jra55do_iaf 1deg_jra55do_ryf_wombatlite

**Pull request merged**